paramgen_src.go

```diff
@@ -1,25 +1,97 @@
 // Code generated by paramgen. DO NOT EDIT.
-// Source: github.com/conduitio/conduit-connector-sdk/cmd/paramgen
+// Source: github.com/ConduitIO/conduit-connector-sdk/tree/main/cmd/paramgen
 
-package connectorname
+package pulsar
 
 import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
 func (SourceConfig) Parameters() map[string]sdk.Parameter {
 	return map[string]sdk.Parameter{
-		"foo": {
+		"connectionTimeout": {
 			Default:     "",
-			Description: "foo is named foo and must be provided by the user.",
+			Description: "connectionTimeout specifies the duration for which the client will attempt to establish a connection before timing out.",
+			Type:        sdk.ParameterTypeDuration,
+			Validations: []sdk.Validation{},
+		},
+		"disableLogging": {
+			Default:     "",
+			Description: "disableLogging is for internal use only",
```

Comment:
can we have more details of what it's useful for

Reply:
That option is more of a development option. By default the golang apache pulsar client logs connection debug messages. I did add it as a parameter option so that I could disable the logs on the integration tests and only see my logs.

Reply:
Pulsar is the first connector I made, I did not see a better alternative.

I could instead do as the kafka connector does:
https://github.com/ConduitIO/conduit-connector-kafka/blob/16fc83a4b1747775189b5917aacbc64a7747dc4f/common/config.go#L46C2-L46C17

It allows setting the logger without exposing that option to the public API. I think the correct change should be that instead, the `disableLogging` shouldn't matter that much to the end user, right?

@maha-hajja what do you think

Reply:
if the user won't use it, you can have it as an unexported field in the config, then paramgen won't expose it as a parameter.
`franzClientOpts` that you linked also does this